pkgs/applications/kde/granatier.nix

```diff
@@ -0,0 +1,17 @@
+{ mkDerivation, lib
+, libkdegames, extra-cmake-modules
+, kdeclarative, knewstuff
+}:
+
+mkDerivation {
+  name = "granatier";
+  meta = with lib; { maintainers = with maintainers; [ freezeboy ]; };
```

Comment:
Please set `description` and `license` fields.

Reply:
Done